cmd/cdi/go.mod

```diff
@@ -1,6 +1,8 @@
 module tags.cncf.io/container-device-interface/cmd/cdi
 
-go 1.19
+go 1.21
```

Comment:
cri-o and containerd are already at 1.21 but podman looks to be at 1.20. So if 1.20 proves to be recent enough for this PR, then it can easily be a better choice. I'll update the PR and then we'll see.

Reply:
Okay, everything looks fine with 1.20.